scripts/create_addon

```diff
@@ -91,6 +91,14 @@ pack_addon() {
   if [ ! -z "$PKG_ADDON_NAME" ] ; then
     ADDON_NAME="$PKG_ADDON_NAME"
   fi
+  for f in $PKG_DIR/repo/screenshot-*.{jpg,png};
+    do if [ -f $f ];
+      then
+        f=$(echo $f | sed 's/.*\///')
```

Comment:
Use `basename` if all you want is the name of the file without the rest of the path.